Rest/Controller/ContentController.php

```diff
@@ -278,4 +285,56 @@ private function getRelation($content, $language)
 
         return false;
     }
+
+    /**
+     * Returns author of the content.
+     *
+     * @param Content $contentValue
+     * @param ContentType $contentType
+     *
+     * @return string
+     */
+    private function getAuthor(Content $contentValue, ContentType $contentType)
+    {
+        $author = $contentValue->getFieldValue($this->getFieldIdentifier('author', $contentType));
+
+        if (null === $author) {
+            $user = $this->contentService->loadContentInfo($contentValue->contentInfo->ownerId);
```

Comment:
`$user` => `$userContentInfo`

Remember to optimize for reading, cause we are going to read this code 200 times more then the 1-2 times you wrote it ;)